src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -205,7 +209,7 @@ const OrderControls = styled.div<{ topAlignDelete: boolean }>`
   display: flex;
   flex-direction: column;
   margin-right: ${size.s};
-  margin-top: ${({ topAlignDelete }) => (topAlignDelete ? "0px" : "20px")};
+  margin-top: ${({ topAlignDelete }) => (topAlignDelete ? "0px" : labelOffset)};
 
   > :not(:last-of-type) {
```

Comment:
(placement of comment is random) For me, `SegmentedControl` still looks strange in some areas of project settings. This is on the patch alias page:

<img width="799" alt="Screen Shot 2022-12-02 at 12 26 41 PM" src="https://user-images.githubusercontent.com/47064971/205350342-5bdfe622-8ca6-414d-b8db-3fb931d3ff58.png">